Containerfile

```diff
@@ -0,0 +1,59 @@
+FROM registry.fedoraproject.org/fedora:41 AS builder
+RUN dnf install -y git-core golang gpgme-devel libassuan-devel && mkdir -p /build/
+ARG GOPROXY=https://proxy.golang.org,direct
+RUN go env -w GOPROXY=$GOPROXY
+COPY . /build
+WORKDIR /build
+# keep in sync with:
+# https://github.com/containers/podman/blob/2981262215f563461d449b9841741339f4d9a894/Makefile#L51
+RUN go build -tags "containers_image_openpgp exclude_graphdriver_btrfs exclude_graphdriver_devicemapper" ./cmd/image-builder
+
+FROM registry.fedoraproject.org/fedora:41
+# Fast-track osbuild so we don't depend on the "slow" Fedora release process to implement new features in bib
+RUN dnf install -y dnf-plugins-core
```

Comment:
Can we put these in a single RUN command, and clean the dnf cache afterwards, to save some disk space and reduce the number of layers?

```
RUN dnf install -y dnf-plugins-core \
  && dnf copr enable -y @osbuild/osbuild \
  && dnf install -y libxcrypt-compat wget osbuild osbuild-ostree osbuild-depsolve-dnf osbuild-lvm2 \
  && dnf clean all
```

Reply:
I've addressed this in a separate commit.